library/std/src/sys/uefi/os.rs

```diff
@@ -0,0 +1,357 @@
+use super::{
+    common::{self, status_to_io_error},
+    unsupported,
+};
+use crate::ffi::{OsStr, OsString};
+use crate::fmt;
+use crate::io;
+use crate::marker::PhantomData;
+use crate::os::uefi;
+use crate::path::{self, PathBuf};
+use crate::{error::Error as StdError, os::uefi::ffi::OsStringExt};
+
+// Return EFI_ABORTED as Status
+pub fn errno() -> i32 {
+    r_efi::efi::Status::ABORTED.as_usize() as i32
```

Comment:
Just a note: SOLID always returns the success value, but maybe this is more appropriate?

Reply:
If `errno` is supposed to return the last error, then I think returning Success might make more sense